polaris-for-stylelint/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@shopify/polaris-for-stylelint",
```

Comment:
https://github.com/Shopify/polaris/issues/5245

Reply:
+1 with @BPScott here. We can call is "Polaris for Stylelint" in a prose context, but I think it would be best to follow the stylelint plugin naming convention when it comes to the package name

Reply:
Updated: https://github.com/Shopify/polaris/pull/5331/commits/cfd3ce753291831827e04af15dac3b13e4db3340